Content.Client/Atmos/Piping/Binary/Systems/GasValveSystem.cs

```diff
@@ -0,0 +1,8 @@
+using Content.Shared.Atmos.Piping.Binary.Systems;
+
+namespace Content.Client.Atmos.Piping.Binary.Systems;
+
+public sealed class GasValveSystem : SharedGasValveSystem
```

Comment:
```suggestion
public sealed partial class GasValveSystem : SharedGasValveSystem
```
makes it easier for forks to add things in separate files

Reply:
I was wrong